docs/product-assembly_migrating-from-BRMS-6.4-to-DM-7.0/src/main/asciidoc/main.adoc

```diff
@@ -1,35 +1,42 @@
-[id='assembly_migrating-from BRMS-6.4-to-DM-{PRODUCT_VERSION}']
+[id='assembly_migrating-from BRMS-6.4-to-DM-7.0']
 
 include::product-shared-docs/document-attributes.adoc[]
 
-= DRAFT Migrating from Red Hat BRMS 6.4 to {PRODUCT} {PRODUCT_VERSION}
+= Migrating from Red Hat BRMS 6.4 to {PRODUCT} {PRODUCT_VERSION}
 // Context attribute is assembly specific and enables module reuse between assemblies.
 :imagesdir: topics/product-shared-docs/images
 :context: installing-dm-on-premise
 include::product-shared-docs/author-group.adoc[]
 
-COMMENT: This document is currently a draft.
-
 // Purpose statement for the assembly
 If you have a Red Hat BRMS 6.4 project that you want to use in {PRODUCT} {PRODUCT_VERSION}, you must first migrate your APIs and data.
 
 .Prerequisite
-You have installed {PRODUCT} {PRODUCT_VERSION}.
+* You have installed {PRODUCT} {PRODUCT_VERSION}, either on premise or on OpenShift.
+* You have a Red Hat BRMS 6.4 installation that contains data and other artifacts that you want to migrate to {PRODUCT} {PRODUCT_VERSION}.
 
 For information about installing {PRODUCT}, see Installing {PRODUCT} on premise or Installing {PRODUCT} on OpenShift. COMMENT: add links.
 
+[NOTE]
+====
+To upgrade Business Planner and address any backwards compatiblity issues, follow the instructions in the https://www.optaplanner.org/download/upgradeRecipe/[*Optaplanner Upgrade Recipe Archive*].
```

Comment:
OptaPlanner (= community) should be called "Business Optimizer" in product 7.x and "Business Resource Planner" in product 6.x.